nuget/rules.bzl

```diff
@@ -0,0 +1,332 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This file is based on the original implementation of https://github.com/SeleniumHQ/selenium/.
```

Comment:
Some things are just stolen (like `dotnet_preamble` and many main ideas), and some things that I'll highlight further are modified. [https://github.com/SeleniumHQ/selenium/](https://github.com/SeleniumHQ/selenium/). They have an Apache-2.0 license.

Reply:
Let's put accredetation in the PR description :)

Reply:
Not sure what accreditation means in this context, but I've mentioned Selenium in the PR =) Hope it's what you was looking for